src/install/browserFetcher.ts

```diff
@@ -122,7 +122,8 @@ function revisionURL(browser: BrowserDescriptor, platform = browserPaths.hostPla
   return util.format(urlTemplate, serverHost, browser.revision);
 }
 
-export async function downloadBrowserWithProgressBar(browserPath: string, browser: BrowserDescriptor): Promise<boolean> {
+export async function downloadBrowserWithProgressBar(browsersPath: string, browser: BrowserDescriptor): Promise<boolean> {
+  const browserPath = browserPaths.browserDirectory(browsersPath, browser);
```

Comment:
I have to admit I'm lost in `browsersPath`, `browserPath`, `browserPaths`. Why would this change?